carbon-design-system/carbon
# chore(project): update examples to parcel 2

This PR updates our examples to use parcel 2. Using parcel 1 was prohibiting us from moving to Yarn 3.1 as it had an install error due to a console message it emitted.

#### Changelog

**New**

**Changed**

- Update parcel 1 examples to parcel 2

**Removed**

#### Testing / Reviewing

- Verify the elements preview works as expected
- Verify you can develop in an examples project that has been changed
